src/themes/journey/theme.json

```diff
@@ -949,6 +949,35 @@
     }
   },
 
+  "richTextBlock": {
+    "base": {
+      "*:first-child": { "mt": 0 }
+    },
+    "panel-primary": {
```

Comment:
should their not be a variants object

Reply:
I did it this way so that in the rich text block in eevee I could pass the variant styling like this: ```sx={{ variant: `richTextBlock.${variant}` }}>``` regardless of if they chose ```base``` or ```panel-primary``` in the drop down. I could also change Contentful to only take variants and not a ```base``` option

Reply:
Yeah to follow convention this should be under a variants object, i.e. `richTextBlock.variants.panel-primary`. As you say this will make the variant selection slightly more complex, but you just need to do something like:
```
x={{ variant: `richTextBlock.${variant === 'base' ? 'base': `variants.${variant}`}` }}
```

Reply:
I've added the variants object in the themes and moved panel-primary under that. I've also updated eevee to reflect the change